footer.php

```diff
@@ -1,9 +1,9 @@
-  <footer>
-    <p>This is our Footer, Here!</p>	
-  </footer>
-
     <?php wp_footer(); ?>
 
+    <footer>
+      <p>This is our Footer, Here!</p>	
+    </footer>
+
   </body>
 
 </html>
```

Comment:
You see this little red icon? As part of standards you need to have an extra new line at the bottom of each file. Then this icon will go away.